lib/resources/example_json/r4_examples_encounter.rb

```diff
@@ -13,6 +13,20 @@ module Resources
         "div": "&lt;div xmlns=\"http://www.w3.org/1999/xhtml\"&gt;&lt;p&gt;&lt;b&gt;Encounter&lt;/b&gt;&lt;/p&gt;&lt;p&gt;&lt;b&gt;Patient&lt;/b&gt;: SMART, JOE&lt;/p&gt;&lt;p&gt;&lt;b&gt;Location&lt;/b&gt;: Baseline East Medical Center, BE Hospital, BE CT&lt;/p&gt;&lt;p&gt;&lt;b&gt;Type&lt;/b&gt;: Inpatient&lt;/p&gt;&lt;p&gt;&lt;b&gt;Service Type&lt;/b&gt;: Emergency Medicine&lt;/p&gt;&lt;p&gt;&lt;b&gt;Class&lt;/b&gt;: inpatient encounter&lt;/p&gt;&lt;p&gt;&lt;b&gt;Status&lt;/b&gt;: In Progress&lt;/p&gt;&lt;p&gt;&lt;b&gt;Period Start Date&lt;/b&gt;: Apr 20, 2016  9:05 P.M. UTC&lt;/p&gt;&lt;p&gt;&lt;b&gt;Reason For Visit&lt;/b&gt;: Abdominal pain&lt;/p&gt;&lt;p&gt;&lt;b&gt;Attending Physician&lt;/b&gt;: Julius, Scott&lt;/p&gt;&lt;/div&gt;"
       },
       "extension": [
+        {
+          "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/custom-attribute",
+          "extension": [
+            {
+             "id": "ENCNTR:17368048",
+             "url": "custom-attribute-name",
+             "valueString": "Full Reg Date/Time"
+            },
+            {
+           "url": "custom-attribute-value",
+             "valueDateTime": "2020-03-04T18:12:22.000Z"
```

Comment:
nit: formatting is slightly off on url and the extensions are only tabbed over a single space.

Reply:
[Fixed](https://github.com/cerner/fhir.cerner.com/pull/438/commits/1e36d2f12646ffbc59fb036010d28b6358218907)